hook-docker/main.go

```diff
@@ -39,19 +40,22 @@ func main() {
 	cmdLines := strings.Split(string(content), " ")
 	cfg := parseCmdLine(cmdLines)
 
-	path := fmt.Sprintf("/etc/docker/certs.d/%s/", cfg.registry)
+	// if tinkServerTLS is not enabled, skip downloading the certs
+	if cfg.tinkServerTLS {
```

Comment:
Should this not be set in `parseCmdLine`?

Reply:
I say we just get rid of this. Fetching the certs is very likely to be done in an unsafe manner and we're better off not making it easy to shoot your foot.